nagios_graphite/main.py

```diff
@@ -4,11 +4,11 @@
 
 from __future__ import print_function
 
-import sys
-import urllib
 import functools
-
 import requests
+import sys
```

Comment:
I prefer to keep builtin imports separate from third-party imports. Any chance we could re-sort these back into that order?